tests/backup/backup_share_test.go

```diff
@@ -608,22 +609,23 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 		Step("Share Backup with Full access to a user of View Only access group and Validate", func() {
 			log.InfoD("Share Backup with Full access to a user of View Only access group and Validate")
 			// Get user from the view access group
-			username, err := backup.GetRandomUserFromGroup(groups[0])
+			var err error
+			chosenUser, err = backup.GetRandomUserFromGroup(groups[0])
```

Comment:
How about chosenUser, err := backup.GetRandomUserFromGroup(groups[0])?

Reply:
`chosenUser` is declared at the test case level not step level. We are using it in multiple steps.